migration-v5.md

```diff
@@ -0,0 +1,316 @@
+# Migration guide
+
+This document serves as a migration guide for `webpack-dev-server@5.0.0`.
+
+## ⚠ BREAKING CHANGES
+
+- Minimum supported `Node.js` version is `14.15.0`.
```

Comment:
let's do it `16` :smile: I think it is time to drop 14 node.js again :smile:

Reply:
```suggestion
- Minimum supported `Node.js` version is `16.10.0`.
```